### PR TITLE
Implement fake data seeding script

### PR DESCRIPTION
**Pull request by blbst123**
- Uses node module faker
- First run:  
DROP DATABASE IF EXISTS photo-gallery; 
CREATE DATABASE hackmo;
- ./database/generateData.js export the functions which will generate data
- In root folder of the project, run "npm run generate-data" in terminal after setting up a database called 'photo-gallery'

### CODE REVIEW

File: database/generateData.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* eslint-disable func-names */
+// Create tables: Images, Users, Restaurants
+const faker = require('faker');
```
Comment (kimbyongjin) [inline database/generateData.js]: Looks great. Very clean and organized.

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+for (let i = 0; i < randomImages.length; i++) {
+  Images.create(randomImages[i])
+    .then((data) => {
+      console.log('Image added to database', randomImages[i].name);
+    });
+}
+
+
```
[the remark this answers is not in the document]
Comment (blbst123) [reply database/index.js]: Thanks! Will do, forgot about it

File: database/generateData.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      comment: faker.lorem.sentence(),
+      date: faker.date.recent(),
+      helpful: Math.floor(Math.random() * 20),
+      not_helpful: Math.floor(Math.random() * 20),
+      restaurant_id: Math.floor(Math.random() * 10)
+    });
+  }
+
+  return arr;
+};
```
[the remark this answers is not in the document]
Comment (hammertube) [reply database/generateData.js]: It is very smart to combine faker and random() method together. It is also nice to make a note that you eslint-disable func-names.

File: package.json [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
+    "generate-data": "node database/index.js"
   },
   "dependencies": {
     "faker": "^4.1.0",
     "jest": "^24.8.0",
     "mysql": "^2.17.1",
+    "mysql2": "^1.6.5",
     "sequelize": "^5.8.6"
   },
   "devDependencies": {
```
[the remark this answers is not in the document]
Comment (hammertube) [reply package.json]: The database index.js is clean and clear. The code speaks for itself.

**Pull request merged**